regext/charter.md

```diff
@@ -1,41 +1,23 @@
-To ensure interoperable implementations it's important to coordinate and standardize EPP extensions.
-
-The registration process of the EPP Extensions Registry is documented in RFC 7451. 
-Extensions may be registered for informational purposes as long as there is a published specification that has been reviewed by a designated expert(s).
 
 The Registration Data Access Protocol (RDAP, STD 95) is the standard for retrieving registration data and metadata from both Domain Name Registries (DNRs) and Regional Internet Registries (RIRs). 
-To ensure interoperable implementations it is important to coordinate and standardize extensions and profiles to be used by registries.
-
-The registration process for extensions to the RDAP is documented in RFC 7480. 
-Extensions may be registered for informational purposes as long as there is a published specification that has been reviewed by a designated expert(s).
-
-Extensions to both protocols that are targeted for the IETF consensus are subject to more thorough review and open discussion within the IETF. 
-In addition, commonality may be discovered in related extensions for which it would make sense to merge them into a single standard extension everybody agrees on.
-
-The REGEXT working group is the home of the coordination effort for EPP and RDAP extensions. 
-The selection of extensions shall incorporate the following guidelines:
-
-Proprietary documented extensions and individual submissions will follow the expert review process as described in the appropriate extensions registry. 
-These extension documents will not be part of the REGEXT working group work or milestones. 
-The working group may discuss or advise on these documents.
 
-* Extensions that seek IETF consensus can be suggested for WG adoption. 
-  If accepted by the working group then the development of the extension may proceed.
+The working group may publish proposed standard and experimental extensions to EPP and RDAP.
+The working group may publish informational and best current practice documents for EPP and RDAP.
+The working group may revise the registration procedures for the EPP and RDAP registries.
+The working group will discuss new registrations with the designated experts for the EPP and RDAP registries.
+When the working group is closed, the mailing list will remain open and available for the use of the expert review process.
 
-* When there are no more proposals for IETF consensus extensions, the working group will either close or become dormant, with the decision made in consultation with the responsible AD. 
-  In any case, the mailing list will remain open and available for the use of the expert review process.
+A primary goal of the working group is to ensure the profiles and extensions are easily discoverable and understood, and to avoid duplicate effort that could harm interoperability.
```

Comment:
It should be safe to simply remove "profiles and" since profiles are forms of extensions.